gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Manage account settings

### Description

Added ability to change user details entered during signup

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1.  signup as a user
2.  goto settings tab
3.  change settings and click save button. Check if a success message shows up. Also, check if the details are changed when you revisit the settings page after saving.
4. Goto settings tab, change a few details. click reset and check if all the changes are gone and form is reloaded with all original details
5. Click change password and enter email id where asked on new a page. Change the password and login again using the new password
6. Click on change email and add a new email address and verify it. You should be able to login using the new email address.
7. Showdown you computer and go back to sleep.

### Miscellaneous Comments

You don't need to remove your email address to be able to add a new email address.

![img 1](https://user-images.githubusercontent.com/38910831/70000974-20b89980-152a-11ea-924d-924b9b259f26.PNG)

![send](https://user-images.githubusercontent.com/38910831/70077950-cffb7c00-15cf-11ea-8d0c-371a28791a3e.PNG)


![img 3](https://user-images.githubusercontent.com/38910831/70000989-2615e400-152a-11ea-856a-6554144cabeb.PNG)


### Checklist
